vtex-sites/starter.store
# chore: add use session example and change tsconfig moduleResolution and module properties to nodenext to allow for @faststore/core/experimental imports.

## What's the purpose of this pull request?

This PR adds support for experimental imports from `@faststore/core/experimental`.

## How does it work?

Changes `moduleResolution` and `module` properties from `tsconfig.json` to `nodenext`. It also adds an example to CustomBuyButton component on how to use `useSession_unstable`. 

## How to test it?

Browse the PDP on the Deploy Preview. You should be able to see the item id followed by the store's country.

### Faststore related PRs

https://github.com/vtex/faststore/pull/2033

## References

https://stackoverflow.com/questions/58990498/package-json-exports-field-not-working-with-typescript
https://www.typescriptlang.org/tsconfig#module
